sygnal/gcmpushkin.py

```diff
@@ -109,8 +119,8 @@ def __init__(self, name, sygnal, config):
             )
 
         self.http_pool = HTTPConnectionPool(reactor=sygnal.reactor)
-        self.max_connections = self.get_config(
-            "max_connections", DEFAULT_MAX_CONNECTIONS
+        self.max_connections = cast(
+            int, self.get_config("max_connections", DEFAULT_MAX_CONNECTIONS)
         )
```

Comment:
This will conceal problems if someone doesn't specify the right thing, better to check it:
```suggestion
        max_connections_raw = self.get_config("max_connections", DEFAULT_MAX_CONNECTION)
        if not isinstance(max_connections_raw, int):
            raise PushkinSetupError("max_connections (if specified) must be an int")
        self.max_connections = max_connections_raw
```